README.md

```diff
@@ -1,2 +1,16 @@
 openmrs-module-chartsearch
 ==========================
+
+## Overview
+
+The Chart Search Module provides an extension to the core application patient search. The module 
+takes advantage of the Apache SOLR search server to index and filter more detailed patient data given precise
+search keywords.
+
+## Installation
+
+To install the Chart Search Module, [download the .omod](https://modules.openmrs.org/#/show/164/chart-search-module), along with its dependencies (OpenMRS 1.9.4) and upload them into your system.
```

Comment:
We may not need to mention 'along with its dependencies (OpenMRS 1.9.4)..' We would inform the user to setup or run the openmrs reference application(which runs all other dependent modules) and then upload or do a normal openmrs module installation for chartsearch, otherwise download openmrs 2.2 + which includes the chartsearch module by default